src/routes/docs/products/databases/permissions/+page.markdoc

```diff
@@ -23,6 +23,10 @@ Configure collection level permissions by navigating to **Your collection** > **
 Document level permissions grant access to individual documents.
 If a user has read, create, update, or delete permissions at the document level, the user can access the **individual document**.
 
+{% info title="Does Appwrite support Row Level Security (RLS)?" %}
+Document level security in Appwrite is similar to what is known as Row Level Security (RLS) found in database systems like PostgreSQL. Both concepts allow you to control access to individual records based on user identity and roles. If you're familiar with RLS, you'll find Appwrite's document level security provides similar granular access control.
```

Comment:
> provides similar granular access control.

While both allows giving granular access, RLS is quite different than Appwrite's permission model so we should make sure that's clear.

Reply:
@stnguyen90 Can you help with the specific difference(s) so we can make it clearer?

Reply:
Postgres' RLS allows you to add a policy to a table. For example, given table:

```sql
CREATE TABLE accounts (manager text, company text, contact_email text);
```

this policy:

```sql
CREATE POLICY account_managers ON accounts TO managers
    USING (manager = current_user);
```

Makes it so when a query is done on the `accounts` table, only rows where the `manager` column matches `current_user` will return.

With Appwrite's document-level permissions, you set on the document which user should have access to that document.

Does that give you enough information to better word it?